src/SonarQubeCypressReporter.js

```diff
@@ -56,6 +58,10 @@ class SonarQubeCypressReporter {
             .attribute('version', 1)
             .element('file');
         this.traverseSuite(node, runner.suite);
+        if (this.totalTestCount === 0) {
```

Comment:
Why don't you just test `runner.suite.tests.length` there instead of a class variable?